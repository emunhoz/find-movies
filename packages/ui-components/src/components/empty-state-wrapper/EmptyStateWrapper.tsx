import React from 'react'
import { EmptyState } from '@commons/images'
import * as S from './EmptyStateWrapper.style'

export const EmptyStateWrapper = () => (
  <S.EmptyStateWrapper>
    <S.EmptyStateImg loading="lazy" width="400" height="200" src={EmptyState} alt="Empty state" />
    <S.EmptyStateTitle>Don&apos;t know what to search?</S.EmptyStateTitle>
    <S.EmptyStateSubTitle>Here&apos;s an offer you can&apos;t refuse</S.EmptyStateSubTitle>
  </S.EmptyStateWrapper>
)
