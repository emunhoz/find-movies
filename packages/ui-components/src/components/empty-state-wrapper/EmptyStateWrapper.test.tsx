import React from 'react'
import { render } from '../../utils/test-utils'
import { EmptyStateWrapper } from './EmptyStateWrapper'

describe('[Component]: EmptyStateWrapper', () => {
  test('render the component', () => {
    const { asFragment } = render(<EmptyStateWrapper />)
    expect(asFragment()).toMatchSnapshot()
  })
})
