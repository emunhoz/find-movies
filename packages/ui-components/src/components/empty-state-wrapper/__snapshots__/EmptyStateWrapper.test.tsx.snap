// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: EmptyStateWrapper render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ hMdVtH"
  >
    <img
      alt="Empty state"
      class="sc-gsTCUz cVyLar"
      height="200"
      loading="lazy"
      src="test-file-stub"
      width="320"
    />
    <h2
      class="sc-dlfnbm jfscyR"
    >
      Don't know what to search?
    </h2>
    <div
      class="sc-hKgILt cQhpCL"
    >
      Here's an offer you can't refuse
    </div>
  </div>
</DocumentFragment>
`;
